FreeRTOS/FreeRTOS-Plus-TCP
# Fix compilation with end point changes

Add changes to fix compilation with End point changes being added on TCP/UDP
Run Uncrusitfy

Test Steps
-----------
As of now compilation test is being done.
```
cmake -S test/build-combination -B test/build-combination/build/ -DTEST_CONFIGURATION=ENABLE_ALL
make -C test/build-combination/build/
```


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
